src/wasm/kotlin/src/arcs/WasmJsInterop.kt

```diff
@@ -109,22 +117,22 @@ fun init(particlePtr: WasmAddress) {
 
 @Retain
 @ExportForCppRuntime("_syncHandle")
-fun syncHandle(particlePtr: WasmAddress, handlePtr: WasmAddress, encoded: WasmString) {
+fun syncHandle(particlePtr: WasmAddress, handlePtr: WasmAddress, encoded: WasmNullableString) {
     log("Getting handle")
     val handle = handlePtr.toObject<Handle>()
-    val encodedStr = encoded.toKString()
-    log("Handle is " + handle.name + "syncing '" + encodedStr + "'")
+    val encodedStr: String? = encoded.toNullableKString()
```

Comment:
Probably don't need the type here, it should be able to be inferred from the result of `toNullableKString` by the compiler.